tests/contrib/BUILD

```diff
@@ -32,3 +33,21 @@ rename_image(
     new_repo = "new_image_repo",
     new_tag = "new_image_tag",
 )
+
+alias(
+    name = "duplicate1",
+    actual = "@distroless_cc//image",
+)
+
+alias(
+    name = "duplicate2",
+    actual = "duplicate1",
+)
+
```

Comment:
can you add one which compares the image against the id as well?

Reply:
Should I just check the id of the pulled image? The test would then break if the image is updated.